prov/util/src/util_attr.c

```diff
@@ -390,6 +390,9 @@ static int fi_resource_mgmt_level(enum fi_resource_mgmt rm_model)
 	}
 }
 
+/* If a provider supports basic registration mode it should set the FI_MR_BASIC
+ * mode bit in prov_mode. Support for FI_MR_SCALABLE is indicated by not setting
+ * any of OFI_MR_BASIC_MAP bits. */
 int ofi_check_mr_mode(uint32_t api_version, uint32_t prov_mode,
 			     uint32_t user_mode)
 {
```

Comment:
Just to improve - Maybe it would be better to use `fi_version()` function instead of `FI_VERSION(1, 5)`. It may help to avoid changes if MINOR/MAJOR version will be stepped up. If you agree with me, can it be replaced in other places as well? I counted about ten places